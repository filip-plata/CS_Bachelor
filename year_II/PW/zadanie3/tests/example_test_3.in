2 1 4
1 x[1] = (3 + x[2])
2 x[0] 5
