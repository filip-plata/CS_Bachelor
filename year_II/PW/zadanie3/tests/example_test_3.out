1 P
2 F
